src/content/docs/browser/browser-integrations/mexn-integration.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: MExN integration
+tags:
+    - New Relic integrations
+metaDescription: Use New Relic browser monitoring to get a dashboard with metrics from your MExN stack.
+---
+
+
+import browserMeanMernIntegtrationDashboard from 'images/browser_screenshot-full_mean-mern-integtration-dashboard.png'
+
+MEAN and MERN stacks are open-source full stack software development kits. Both kits use a single codebase to develop web applications. New Relic has combined both stacks into a single quickstart. Our MExN integration makes use of the Browser agent, Node,js agent, and MongoDB integration to provide you with a pre-built dashboard with your most important metrics (like response time, CPU utilization, traffic and login frequencies).
+
+<img
+  title="Mean Mern dashboard"
+  alt="A screenshot depicting the mean mern dashboard"
+  src={browserMeanMernIntegtrationDashboard}
+/>
+
+<figcaption>
+After setting up our MExN integration, we give you a dashboard for your MExN web app metrics. 
+</figcaption>
+
+## Install [#install]
+
+### Step 1 (optional): Install the infrastructure agent [#infra-install]
+
+Installing the infrastructure agent is optional, but doing this will get you important metrics about the host where your app is running. 
+
+To do this, follow [the infrastructure agent install steps](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent).
+
+### Step 2: Install MongoDB quickstart
+
+1. Check out our MongoDB agent requirements in our [documentation](/docs/infrastructure/host-integrations/host-integrations-list/mongodb-monitoring-integration/)  before installing the agent.
+2. Open the [MongoDB quickstart installation](https://newrelic.com/instant-observability/mongodb). 
+3. Click  **install now** at the bottom of the page to start the MongoDB agent installation.
+
+### Step 3: Install Node.js agent
+
+1. Check out our Node.js agent requirements in our [documentation](/docs/apm/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent/).
+2. Open the [Node.js quickstart installation](https://newrelic.com/instant-observability/node-js). 
+3. Click  **install now** at the bottom of the page to start the Node.js agent installation.
+
+### Step 4: Install the browser agent [#browser-install] 
+
+To install our browser agent: 
+
+1. From [one.newrelic.com](https://one.newrelic.com), go to the [**Add data** page](https://one.newrelic.com/marketplace). 
+2. Click **Browser and mobile**, then click **Browser monitoring**.
+3. Select the **Select copy/paste JavaScript code** option (for more about this code, see [our browser install docs](/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent)). 
+4. Complete the rest of the steps and click **Enable.**
+5. Go the `index.html ` file of your React application.
```

Comment:
5. Go the `index.html ` file of your React or Angular application.

Reply:
done!